tracer/build/_build/Build.Steps.cs

```diff
@@ -1360,6 +1378,7 @@ void RunWindowsIisIntegrationTests(Project project)
                         .SetLogsDirectory(TestLogsDirectory)
                         .When(TestAllPackageVersions, o => o.SetProcessEnvironmentVariable("TestAllPackageVersions", "true"))
                         .When(IncludeMinorPackageVersions, o => o.SetProperty("IncludeMinorPackageVersions", "true"))
+                        .When(IncludeTestsRequiringDocker is not null, o => o.SetProperty("IncludeTestsRequiringDocker", IncludeTestsRequiringDocker.Value ? "true": false))
```

Comment:
feels weird to have a string and a bool in the same ternary operation

Reply:
haha, yeah, oops